Sources/SBTUITestTunnelServer/SBTUITestTunnelServer.m

```diff
@@ -1211,7 +1211,18 @@ - (NSDictionary *)commandCoreLocationStubAccuracyAuthorization:(NSDictionary *)p
             
             [CLLocationManager setStubbedAccuracyAuthorization:accuracyAuthorization];
             for (CLLocationManager *locationManager in self.coreLocationActiveManagers.keyEnumerator.allObjects) {
-                [locationManager.stubbedDelegate locationManagerDidChangeAuthorization:locationManager];
+                switch (locationManager.authorizationStatus) {
+                    case kCLAuthorizationStatusAuthorizedAlways:
+                    case kCLAuthorizationStatusAuthorizedWhenInUse: {
+                            dispatch_async(dispatch_get_main_queue(), ^{
+                                [locationManager.stubbedDelegate locationManagerDidChangeAuthorization:locationManager];
```

Comment:
Just for my knowledge, why are we limiting the propagation of `locationManagerDidChangeAuthorization` only for these two status values?

Reply:
I guess I copy pasted too much here